app/src/pages/ProfileEdit.tsx

```diff
@@ -1,130 +1,117 @@
-import * as React from "react"
-import QuestionField from "../components/ProfileEdit/QuestionField"
+import * as React from 'react'
+import QuestionPage from '../components/ProfileEdit/QuestionPage'
 
-import styled from "styled-components"
-import { Button, LinearProgress } from '@material-ui/core';
-import { Question } from "../models/Question"
+import styled from 'styled-components'
 
 const Container = styled.div`
-  margin: 30px auto;
-  padding: 0 15px;
-  max-width: 1140px;
+    margin: 30px auto;
+    padding: 0 15px;
+    max-width: 1140px;
 `
 
-const StyledButton = styled(Button)`
-  font-size: 16px;
-  margin: 5px !important;
-`
-
-const questions = [
-  {
-    id: '1',
-    question: 'How would you describe your home?',
-    type: 'radio',
-    options: [
-      {label: 'Owned Single-Unit', value: 'single'}, 
-      {label: 'Owned Multi-Unit', value: 'multi'},
-      {label: 'Owned House', value: 'house'}
-    ],
-    answer: 'house'
-  },
-  {
-    id: '2',
-    question: 'Do you allow drinking there?',
-    type: 'radio',
-    options: [
-      {label: 'Yes', value: 'yes'}, 
-      {label: 'We don\'t drink, but it is allowed', value: 'we-dont'}, 
-      {label: 'No', value: 'no'}
-    ],
-    answer: 'yes'
-  },
-  {
-    id: '3',
-    question: 'Do you allow smoking at your residence?',
-    type: 'radio',
-    options: [
-      {label: 'Yes, we smoke inside', value: 'inside'},
-      {label: 'Yes, but only outside', value: 'outside'},
-      {label: 'No', value: 'no'}
-    ],
-    answer: 'outside'
-  },
-  {
-    id: '4',
-    question: 'Do you allow substance use at your residence?',
-    type: 'radio',
-    options: [
-      {label: 'Yes', value: 'yes'},
-      {label: 'No', value: 'no'},
-    ],
-    answer: 'no'
-  },
-  {
-    id: '5',
-    question: 'What are your interests?',
-    type: 'checkbox',
-    options: [
-      {label: 'Tinkering', value: 'tinkering'},
-      {label: 'Trashy TV', value: 'tv'},
-      {label: 'Puzzles', value: 'puzzles'},
-      {label: 'Cheesecakes', value: 'cheesecakes'}
-    ],
-    answer: {
-      tinkering: false,
-      tv: true,
-      puzzles: false,
-      cheesecakes: true,
-    }
-  },
-  {
-    id: '6',
-    question: 'Tell us about yourself',
-    type: 'textarea',
-    answer: 'I have many leather-bound books'
-  }
-];
-
-const initialState = {
+const getQuestions = (org: string, section: string) => {
```

Comment:
this will be replaced with data retrieved from API, but can we save this artifact somewhere with meaning name? The 'data' folder would be appropriate, when the time comes. This can be useful to maintain for future unit testing